appinventor/appengine/src/com/google/appinventor/client/editor/simple/components/MockListView.java

```diff
@@ -293,7 +293,7 @@ private void createImageTwoTextLayout(JSONObject object) {
   }
 
   private void setItemHeight(boolean detail, boolean image) {
-    int mainHeight = Integer.parseInt(mainFontSize);
+    int mainHeight = (int) Float.parseFloat(mainFontSize);
```

Comment:
This is a niggle, but if we need an integer for the the height of the text item, it seems better to use Math.Ceil(). That is the math method that rounds a float up to the next larger int. The int cast just chops of the decimal, so we'd end up with a text item height smaller than the font height.

DetailHeight already uses the casting method, so it seems to work reasonably the way it is.